tape/src/main/java/com/squareup/tape/CorruptionException.java

```diff
@@ -0,0 +1,12 @@
+package com.squareup.tape;
+
+import java.io.IOException;
+
+/**
+ * Exception class representing corruption in the Tape queue.
+ */
+public class CorruptionException extends IOException {
```

Comment:
Doesn't Tape  already have a subclass of IOException for all of its errors? If yes you could subclass it or pass a custom error message.

Reply:
It does have it's own exception - FileException but that converts it later on as it extends a RuntimeException.